tests/components/dynalite/test_config_flow.py

```diff
@@ -3,33 +3,27 @@
 
 from homeassistant.components.dynalite import config_flow
 
-from tests.common import mock_coro
+
+async def test_configured_hosts():
+    """Test the configured_host method."""
+    hass = Mock()
+    entry = Mock()
+    host = "abc"
+    entry.data = {"host": host}
+    hass.config_entries.async_entries = Mock(return_value=[entry])
```

Comment:
Don't patch the core. Use `MockConfigEntry` from tests/common.py and add a mock entry to hass. Then start the flow via core `hass.config_entries.flow.async_init`. We can probably remove `configured_hosts` helper when using the unique_id config flow api.

Reply:
done